wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/registry/LanguageServerRegistryImpl.java

```diff
@@ -189,8 +212,16 @@ public ServerCapabilities initialize(String fileUri) throws LanguageServerExcept
     if (language == null) {
       return Collections.emptyList();
     }
+    List<LanguageServerLauncher> combinedLaunchers = new LinkedList<>(launchers);
+    Workspace workspace = getWorkspaceConfiguration();
```

Comment:
I'm not an expert in this area of Che, but by looking at a method argument `fileUri` I suppose that this method can be called a lot of times because of different files. If this is true then looks like this code might call Workspace API a lot of times. Wouldn't it be better to cache GET WS API calls results?